src/NWAClient.ts

```diff
@@ -0,0 +1,249 @@
+import { bytesToHex, hexToBytes } from "@noble/hashes/utils";
+import { generateSecretKey, getPublicKey, Relay } from "nostr-tools";
+import {
+  BudgetRenewalPeriod,
+  Nip47Method,
+  Nip47NetworkError,
+  Nip47NotificationType,
+  NWCClient,
+} from "./NWCClient";
+
+export type NWAOptions = {
+  relayUrl: string;
+  appSecretKey?: string;
+  appPubkey: string;
```

Comment:
Can we arrange this for readability so that required fields are on top? (Also great if the same order is followed in `NewNWAClientOptions`)

Reply:
Changing to `export type NewNWAClientOptions = Omit<NWAOptions, "appPubkey">;` to remove duplication